### PR TITLE
Ndefries/epidatr-slides

### PR DESCRIPTION
**Pull request by nmdefries**
Slides about `epidatr`

@dshemetov feel free to liberally make changes here -- a lot of the text is just copied from the README, etc

### CODE REVIEW

File: index.qmd [hunk 1]
```diff
@@ -15,5 +15,115 @@ execute:
...
+```
+
+## API key setup
+
+The Delphi API requires a (free) API key for full functionality. To generate
+your key, register for a pseudo-anonymous account
+[here](https://api.delphi.cmu.edu/epidata/admin/registration_form) and see more
+discussion on the [general API
+website](https://cmu-delphi.github.io/delphi-epidata/api/api_keys.html). See the
+`set_api_key()` function documentation for details on how to use your API key.
```
Comment (nmdefries) [inline index.qmd]: I copied this from the readme on `dev`, but realized that `set_api_key` isn't available in the CRAN version of epidatr. We probably don't want to give the [old API key advice](https://github.com/cmu-delphi/epidatr/blob/0adb796a54aee22cc54f9e1d0a724492892267b9/vignettes/epidatr.Rmd#L31-L33), though.

Comment (dshemetov) [reply index.qmd]: Right, I'm thinking we just give the new advice for `save_api_key()`. Hopefully we can get it to CRAN before Tuesday and if not, we'll get it soon enough after the meeting. Really eager users can probably figure out how to use environment variables on their own.

Comment (nmdefries) [reply index.qmd]: Sounds good, thanks!

**Comment by brookslogan** [comment]
Thanks for preparing these slides!  Look pretty good to me; a few questions + a note:
- which version of epidatr are we on for the presentation? stable or dev? any recommendation between the two?
- 7dav signal... is that a good choice?  Does it still have holes in its history or other issues? (@dshemetov maybe?) 
- for forming an `epi_archive`, we should never use an issue query that starts midway through the history of the data source unless we also pair it with an as_of query to get the full snapshot at or immediately before the first issue in that range.  I can take care of editing this, just a heads up.

**Comment by dshemetov** [comment]
- I think `dev` should be safe to recommend. Mostly contains documentation fixes, a few bugfixes, and the auth workflow changes.
- chose 7dav because it was in the docs, we can swap to something non-derived
- good call, just didn't want to download issue=*, but fixing with as_of on the left end is a really nice idea, thanks

**Comment by brookslogan** [comment]
(Re. the issue range, I normally end up using an extreme value like 12340101 for the LHS of an epirange so that I can inspect and verify whatever assumptions would lead me to want to filter the as_ofs of interest and just work on the larger archive.  [And extreme values in both time ranges and issue ranges are well-supported by epidatr; it's not looping queries based on the time range like covidcast.])

**Comment by dshemetov** [comment]
Makes sense, I just wasn't sure if the hhs flu data goes all the way back to 2000's or 2020. Didn't want to accidentally download 20 years of issues.

**Pull request merged**